internal/manager.go

```diff
@@ -384,22 +385,24 @@ func (m *Manager) downloadPlugin(downloadUrl string) (string, error) {
 }
 
 func (m *Manager) Add(pluginName, url, alias string) error {
+	// For compatibility with older versions of plugin names <category>/<plugin-name>
+	if strings.Contains(pluginName, "/") {
```

Comment:
While this is unlikely to be the case with custom plugins, it's harmless enough to keep.